### ISSUE TITLE
Predicts only background for lung tumour segmentation

### ISSUE THREAD

**Issue by trathpai** [ellisdg/3DUnetCNN#75]
First of all nice work implementing the U-net . I was trying to make the code work for lung tumour segmentation using isensee model. But it only predicts background in test set. I have 3 labels 0 as background 1 as lung and 2 as tumour in the lung.  Any idea how to set the weights for dice coefficients and should i include 0 as label in the config dictionary?

Thanks!!

**Comment by ellisdg** [comment]
I can't tell you what is going wrong, but don't include 0 in the labels. I have it setup so that the default is 0. What does your training/validation loss look like?

**Comment by trathpai** [comment]
Thanks for replying , train validation curve look fine to me, seems to converge. But dice coefficient is very poor since its a CT volume and i need to preprocess the volumes.

**Comment by Pmac23** [comment]
Hi I have kind of a similar issue with poor dice score with vessel segmentation in  3d volumes. @trathpai did you find a solution to improve the same?
